lib/fastlane/plugin/firebase_app_distribution/client/firebase_app_distribution_api_client.rb

```diff
@@ -71,16 +71,17 @@ def update_release_notes(release_name, release_notes)
               text: release_notes
             }
           }
-          connection.patch(update_release_notes_url(release_name), payload.to_json) do |request|
+          response = connection.patch(update_release_notes_url(release_name), payload.to_json) do |request|
             request.headers[AUTHORIZATION] = "Bearer " + @auth_token
             request.headers[CONTENT_TYPE] = APPLICATION_JSON
             request.headers[CLIENT_VERSION] = client_version_header_value
           end
+          UI.success("✅ Posted release notes.")
+          response.body
```

Comment:
1. This method still supports nil or empty release notes, but it returns nil in that case. Since we're now always going to pass in notes I think we should stop handling that case.
2. We should update the documentation for this method to specify the new return value, and any other changes.

Reply:
1. Good call! Forgot to remove that code. Done.
2. Done.